### PR TITLE
implementing fake_quantize_per_channel_affine

### PR DESCRIPTION
**Pull request by MovGP0**
this PR contains the following changes:

- implementation of `torch.fake_quantize_per_channel_affine`
- cleanup of `AdjustGamma` class

### CODE REVIEW

**Comment by NiklasGustafsson** [comment]
Error in all builds:

```C#
test\TorchSharpTest\TestTorchTensor.cs(7771,19): error CS0623: Array initializers can only be used in a variable or field initializer. 
```

**Comment by NiklasGustafsson** [comment]
> Remark: I should build and run tests before committing. However, my local build still doesn't work all the time, so using the CI is my workaround for now 😖

Understood. Here are test failures:

```
Failed TorchSharp.TestTensor.TestFakeQuantizePerChannelAffine [22 ms]
  Error Message:
   System.Runtime.InteropServices.ExternalException : !needs_dynamic_casting<func_t>::check(iter) INTERNAL ASSERT FAILED at "../aten/src/ATen/native/cpu/Loops.h":308, please report a bug to PyTorch. 
Exception raised from cpu_kernel at ../aten/src/ATen/native/cpu/Loops.h:308 (most recent call first):
```

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,96 @@
+{
```
Comment (NiklasGustafsson) [inline .vscode/settings.json:1]: This looks like a local VS Code file. should it really have been checked in?

Comment (MovGP0) [reply .vscode/settings.json:1]: no. it shouldn't. Will remove the file and add an entry to `.gitignore`.

Haven't found the time to fix the issue with the unit test yet :-/ Trying to analyze & fix it next weekend.

**Comment by NiklasGustafsson** [comment]
Sorry, I had to make a release of 0.99.5 today.

**Comment by NiklasGustafsson** [comment]
@MovGP0 -- Ping! Is this ready to merge? It'd be great to get a little one-line blurb on this change for the release notes.

File: test/TorchSharpTest/TestTorchTensor.cs [hunk 2]
```diff
@@ -7804,6 +7810,17 @@ public void TestHistogram()
...
 
+        [Fact]
+        [TestOf(nameof(torch.fake_quantize_per_channel_affine))]
+        public void TestFakeQuantizePerChannelAffine()
+        {
+            var x = torch.rand(2, 2, 2);
+            var scales = (torch.randn(2) + 1d) * 0.05d;
+            var zero_points = torch.zeros(2).to(torch.int32);
+            var result = torch.fake_quantize_per_channel_affine(x, scales, zero_points, axis: 0, quant_min: 0, quant_max: 255);
+            Assert.True(true);
```
Comment (NiklasGustafsson) [inline test/TorchSharpTest/TestTorchTensor.cs:7821]: Huh? Is this just because the test is really just looking for an exception in the call?

**Comment by MovGP0** [comment]
> Sorry, I had to make a release of 0.99.5 today.

Hey, sorry that I reply this late. Have the championships this and next month and also a new job, so spending all my free time with training or learning 😔 Will probably find time in 4 weeks for further contributions 😊

File: test/TorchSharpTest/TestTorchTensor.cs (hunk 2 above)
Comment (MovGP0) [reply test/TorchSharpTest/TestTorchTensor.cs:7821]: Took this code from the official documentation. It's the minimal amount of testing. 

Should test the result values, but would need to get rid of the random initializer.

Comment (NiklasGustafsson) [reply test/TorchSharpTest/TestTorchTensor.cs:7821]: Either use a Generator instance, or (I do this frequently) just create a tensor from the random values in the PyTorch doc and then the correct results should be right there in the documentation already...

**Comment by NiklasGustafsson** [comment]
@MovGP0 -- do you want to keep this PR open?

**Pull request closed without merging**